model-engine/model_engine_server/api/llms_v1.py

```diff
@@ -238,6 +238,29 @@ async def event_generator():
             except InvalidRequestException as exc:
                 yield {"data": {"error": {"status_code": 400, "detail": str(exc)}}}
                 return
+            except (ObjectNotFoundException, ObjectNotAuthorizedException):
+                yield {
+                    "data": {
+                        "error": {
+                            "status_code": 404,
+                            "detail": "The specified endpoint could not be found.",
+                        }
+                    }
+                }
+                return
+            except ObjectHasInvalidValueException as exc:
+                yield {"data": {"error": {"status_code": 400, "detail": str(exc)}}}
+                return
+            except EndpointUnsupportedInferenceTypeException as exc:
+                yield {
+                    "data": {
+                        "error": {
+                            "status_code": 400,
+                            "detail": f"Unsupported inference type: {str(exc)}",
```

Comment:
Would be ergonomic to tell the user what type they specified, and what the valid values are. Or does the inner exception tell them that?

Reply:
The inner error message tells them something more specific, i.e. 

`"Endpoint {model_endpoint_name} is not a streaming endpoint.`

https://github.com/scaleapi/llm-engine/blob/b8db3eab78ebd67e9c4f427b0a792e36598f24f1/model-engine/model_engine_server/domain/use_cases/llm_model_endpoint_use_cases.py#L1003-L1006

Reply:
i think there errors are thrown at `use_case.execute` and not ever in `event_generator`?

Reply:
oh man, I think I tested this incorrectly earlier, the error handling actually seems fine for the `use_case` level on ObjectNotFound and EndpointUnsupported. I'm gonna close this pr.